src-qt5/core-utils/lumina-config/i18n/lumina-config_sv.ts

```diff
@@ -1147,12 +1147,12 @@
     <message>
         <location filename="../pages/page_compton.ui" line="32"/>
         <source>Disable Compositing Manager (session restart required)</source>
-        <translation type="unfinished"></translation>
+        <translation>unfinished</translation>
```

Comment:
This needs to be finished, or reverted back to having the `type="unfinished"` tag.